README.md

```diff
@@ -216,7 +216,7 @@ servers       : {@{id=9cabadffe05cbdaedd3cee7ef763956f; host=AC1SRV0004; os=; up
 ```powershell
 PS > $params = @{
         level= "level_critical"
-        exploitable= "true"
+        exploitable = "true"
```

Comment:
No, here you set our standard lower.

```
# Do not write:
$variable=Get-Content $FilePath -Wai:($ReadCount-gt0) -First($ReadCount*5)

# Instead write:
$variable = Get-Content -Path $FilePath -Wait:($ReadCount -gt 0) -TotalCount ($ReadCount * 5)
```
source : https://poshcode.gitbooks.io/powershell-practice-and-style/content/Style-Guide/Code-Layout-and-Formatting.html